### DISCUSSION TITLE
Can I / Should I keep frigate on subnet without internet access

### DISCUSSION THREAD

**Discussion by wisemonkey** [blakeblackshear/frigate#6513] [General]
Basically I'm trying to decide whether to keep frigate on subnet without internet access or is it enough to have only cameras on such a network.
Mainly from security and data leak point of view.

If frigate is kept on subnet without internet then is there way for updates?

**Comment by blakeblackshear** [comment]
I run Frigate on a network that has internet access and I connect my cameras to a second NIC on the server that has a static IP. The cameras don't have internet access, but Frigate does.

**Reply by wisemonkey** [reply]
Thanks that is what I'm leaning towards right now
